CHANGES.md

```diff
@@ -18,6 +18,7 @@
 * Reread repair configuration when a node state changes - Issues #470 and #478
 * Support configuring backoff for failed jobs - Issue #475
 * Dropping keyspaces does not clean up schedules - Issue #469
+* Make lock refresh + failure cache configurable - Issues #543 and #587
```

Comment:
Remove "and 587", 587 is the PR number. It shouldn't be in the changelog.
Also move this to the top, latest changes should always be at the top.